system-parachains/asset-hubs/asset-hub-kusama/src/genesis_config_presets.rs

```diff
@@ -27,41 +27,50 @@ fn asset_hub_kusama_genesis(
 	endowed_accounts: Vec<AccountId>,
 	id: ParaId,
 ) -> serde_json::Value {
-	serde_json::json!({
-		"balances": BalancesConfig {
+	let config = RuntimeGenesisConfig {
+		system: Default::default(),
+		balances: BalancesConfig {
 			balances: endowed_accounts
 				.iter()
 				.cloned()
 				.map(|k| (k, ASSET_HUB_KUSAMA_ED * 4096 * 4096))
 				.collect(),
 		},
-		"parachainInfo": ParachainInfoConfig {
-			parachain_id: id,
-			..Default::default()
-		},
-		"collatorSelection": CollatorSelectionConfig {
+		transaction_payment: Default::default(),
+		parachain_info: ParachainInfoConfig { parachain_id: id, ..Default::default() },
+		collator_selection: CollatorSelectionConfig {
 			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
 			candidacy_bond: ASSET_HUB_KUSAMA_ED * 16,
 			..Default::default()
 		},
-		"session": SessionConfig {
+		session: SessionConfig {
 			keys: invulnerables
 				.into_iter()
 				.map(|(acc, aura)| {
 					(
-						acc.clone(),                         // account id
-						acc,                                 // validator id
-						SessionKeys { aura }, 	// session keys
+						acc.clone(),          // account id
+						acc,                  // validator id
+						SessionKeys { aura }, // session keys
 					)
 				})
 				.collect(),
 		},
-		"polkadotXcm": {
-			"safeXcmVersion": Some(SAFE_XCM_VERSION),
+		aura: Default::default(),
+		aura_ext: Default::default(),
+		polkadot_xcm: PolkadotXcmConfig {
+			_config: Default::default(),
+			safe_xcm_version: Some(SAFE_XCM_VERSION),
 		},
 		// no need to pass anything to aura, in fact it will panic if we do. Session will take care
```

Comment:
This comment needs to go to line 58, at this position it does not make so much sense.